easybuild/easyconfigs/s/SAMtools/SAMtools-1.2-foss-2015b.eb

```diff
@@ -0,0 +1,33 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Robert Schmidt <roschmidt@ohri.ca>, Cedric Laczny <cedric.laczny@uni.lu>, Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+##
+
+name = 'SAMtools'
+version = '1.2'
+
+homepage = 'http://www.htslib.org/'
+description = """SAM Tools provide various utilities for manipulating alignments in the SAM format, 
+ including sorting, merging, indexing and generating alignments in a per-position format."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+sources = [SOURCELOWER_TAR_BZ2]
+source_urls = ['https://github.com/samtools/%(namelower)s/releases/download/%(version)s']
+
+patches = ['SAMtools-1.1_Makefile.patch']
+
+dependencies = [
+    ('ncurses', '6.0'),
```

Comment:
same remark here as in another PR, it's probably better to stick with ncurses 5.9 for now to avoid conflicts when this is loaded together with other modules

we plan to switch to ncurses 6.0 for the `2016a` toolchains (soon to be defined)